lib/schema/facter.yaml

```diff
@@ -178,6 +178,12 @@ dhcp_servers:
             type: ip
             description: The DHCP server for the default interface.
 
+dir:
```

Comment:
We should fix this on Windows. @Iristyle it comes in as a result of https://github.com/puppetlabs/puppet_for_the_win/blob/master/conf/windows/stage/bin/environment.bat#L20.